### PR TITLE
Req Passive Income

### PR DESCRIPTION
**Pull request by Hathkar**
## About The Pull Request

Reduces requisitions passive point income to 2 points per minute, down from 4.

## Why It's Good For The Game
Marines now get fewer points for turtling up and doing nothing. This will create more incentive to push out and secure mining drills, xeno corpses, and intel packages (#4369).

## Changelog
:cl:
balance: Passive Point Income
/:cl:

### CODE REVIEW

**Comment by psykzz** [comment]
Removing the scaling is a bad idea since it just removes the scaling with pop. What if we get 100 players? 
We already get ~60 players so typically squad points are doubled at round start.

**Comment by spookydonut** [comment]
Don't remove round start point scaling that will massively destroy balance between low/high pop.

**Comment by Hathkar** [comment]
Reverted the scaling change.

**Comment by RenGusta** [comment]
I disagree with the necessity of this PR until we see the impact of the other two Req Impacting PRs you have anyway. 

I'd like if you could refrain from testing this until we see the effect of the other two PRs tanking req impact.

File: code/datums/gamemodes/distress.dm [hunk 1]
```diff
@@ -222,6 +222,7 @@
 
 	var/scale = max(marine_pop_size / MARINE_GEAR_SCALING, 1) //This gives a decimal value representing a scaling multiplier. Cannot go below 1
 
+
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm:225]:
```suggestion
```

File: code/controllers/subsystem/points.dm [hunk 2]
```diff
@@ -77,7 +77,7 @@ SUBSYSTEM_DEF(points)
 
 /datum/controller/subsystem/points/proc/scale_supply_points(scale)
 	supply_points = round(supply_points * scale)
-
+
```
Comment (Rohesie) [inline code/controllers/subsystem/points.dm:80]:
```suggestion

```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-2). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**